src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -124,18 +141,16 @@ public boolean login(final String userName, final HashedPassword hashedPassword)
         }
       }
       // update last login time
-      try (
-          final PreparedStatement ps = con.prepareStatement("update ta_users set lastLogin = ? where username = ?")) {
-        ps.setTimestamp(1, new Timestamp(System.currentTimeMillis()));
+      try (final PreparedStatement ps = con.prepareStatement("update ta_users set lastLogin=? where username=?")) {
+        ps.setTimestamp(1, Timestamp.from(Instant.now()));
         ps.setString(2, userName);
         ps.execute();
         con.commit();
         return true;
       }
     } catch (final SQLException sqle) {
       throw new IllegalStateException(
-          "Error validating password name:" + userName + " : " + " pwd:" + hashedPassword.mask(),
-          sqle);
+          String.format("Error validating password name: '%s' pwd: %s", userName, hashedPassword.mask()), sqle);
```

Comment:
while this is being updated, dumping even the hashed password is not good practice. Would you mind removing the password from the error message?

Reply:
We are not printing the hashed password, we are just printing *********** in the length of the hashed password (this helped me fixing some integTests)